homeassistant/components/iotty/const.py

```diff
@@ -0,0 +1,19 @@
+"""Constants for the iotty integration."""
+
+from datetime import timedelta
+import logging
+
+from homeassistant.const import Platform
+
+DOMAIN = "iotty"
+
+OAUTH2_AUTHORIZE = "https://auth.iotty.com/.auth/oauth2/login"
+OAUTH2_TOKEN = "https://auth.iotty.com/.auth/oauth2/token"
+OAUTH2_CLIENT_ID = "hass-iotty"
+
+IOTTYAPI_BASE = "https://homeassistant.iotty.com/"
+
+PLATFORMS: list[Platform] = [Platform.SWITCH]
+
+LOGGER = logging.getLogger(__package__)
+UPDATE_INTERVAL = timedelta(seconds=30)
```

Comment:
Please move this constant, which is only used in `coordinator.py` to `coordinator.py`.

Same comment for other constants which are only used once.

Reply:
Hi, the way you are suggesting is the way it was originally. During the review, @edenhaus actually suggested moving it into the consts, and so we did.

Reply:
Constants which are used only in one module should not be in const.py. Constants which are used more than once should be in const.py, I just double checked and @edenhaus agrees. Maybe there was some misunderstanding.